src/map/status.cpp

```diff
@@ -8733,7 +8733,7 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 			return 0;
 	}
 
-	rate = cap_value(rate, 0, 10000);
+	rate = max(0, rate);
```

Comment:
This doesn't fully fix the issue. If you equip stuff that has an added chance of over 100%, it will still go with 100% into this method.

I also noticed another issue we added here: https://github.com/rathena/rathena/commit/2993c0baa64ec41181d0cf465635f988e12f3893

We should not give sc_def2 a lower cap as that breaks how pre-renewal chances work (you can increase chance if you're higher level than target).
Alternatively you can add a lower cap only for renewal and leave it as it was before for pre-renewal.